docker/scripts/cyber_start.sh

```diff
@@ -492,6 +492,7 @@ function start_cyber_container() {
         --hostname "${CYBER_INSIDE}" \
         --shm-size 2G \
         --pid=host \
+        --device /dev/snd \
```

Comment:
There is already a setup_device functin in `scripts/apollo_base.sh`.

And all /dev on host is mounted into docker. Cf.
https://github.com/ApolloAuto/apollo/blob/master/docker/scripts/cyber_start.sh#L364

Reply:
Sorry, did not notice this. I will remove this line